src/components/common/Button/ButtonIconAlignment.tsx

```diff
@@ -0,0 +1,3 @@
+export type ButtonIconAlignment = 'left' | 'right'
+
+export const ButtonIconAlignmentValues = ['left', 'right']
```

Comment:
```suggestion
export const ButtonIconAlignment = ['left', 'right'] as const

export type ButtonIconAlignment = typeof ButtonIconAlignment[number]
```

Reply:
Sorry, I overlooked this one. I'll fix it.